### PR TITLE
Added questions to guide the learner through each section

### PR DESCRIPTION
**Pull request by Nico-Sanchez**
* Added questions to guide the learner knowledge (in spanish)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,233 +1,299 @@
 # LambdaClass Hacking Learning Path Handbook
-
+
```
Comment (unbalancedparentheses) [inline README.md:2]: @Nico-Sanchez I think you need to change your coding editor config. you are adding a new char for every line of the file. please change your editor's config and resubmit the PR. Thanks!

**Comment by unbalancedparentheses** [comment]
PD: Pleade add a more descriptive title to the PR @Nico-Sanchez

**Review by fkrause98 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Nico-Sanchez) [reply README.md:2]: It's not my coding editor config, we were working in google docs and copied pasted here (horrible choice) and it added a new char every line.

**Comment by Nico-Sanchez** [comment]
Closing because we already have another PR with this PR updated work.

**Pull request closed without merging**